tools/downloader/src/open_model_zoo/model_tools/_configuration.py

```diff
@@ -13,155 +13,20 @@
 # limitations under the License.
 
 import collections
-import contextlib
 import fnmatch
 import re
 import shlex
-import shutil
 import sys
-
-from pathlib import Path
-
-import requests
 import yaml
 
 from open_model_zoo.model_tools import _common
+from open_model_zoo.model_tools.download_engine import file_source, postprocessing, validation
```

Comment:
Looks like FileSource is enough here, but whole module is imported. Is it to state module attachment in the code?